dev/openmage/nginx-frontend.conf

```diff
@@ -0,0 +1,59 @@
+server {
+    listen 80;
+    server_name _ "";
+
+    if ( $request_method !~ ^(GET|POST|HEAD|OPTIONS)$) { return 405; }

```

Comment:
What's wrong with PUT, PATCH (not implemented anywhere in M1), and DELETE?

Reply:
It's probably not necessary to block them, but why allow them?

Reply:
Because APIs need them.

Reply:
I updated the admin config. For the frontend I don't intend on adding support for the API as I think it is safer to leave it disabled on the more public sites and only leave it enabled on the admin site which *should* be on a separate domain and perhaps behind some sort of more restrictive WAF.